src/services.ts

```diff
@@ -0,0 +1,185 @@
+import { LoginProps } from './components/Login/Login.types';
+import { RegistrationProps } from './components/Registration/Registration.types';
+import { CourseType } from './store/courses/types';
+import { AuthorType } from './store/authors/types';
+
+export function login(name: string, email: string, password: string) {
+	const data: LoginProps = {
+		name: name,
+		email: email,
+		password: password,
+	};
+	fetch('http://localhost:4000/login', {
+		method: 'POST',
+		headers: new Headers({ 'content-type': 'application/json' }),
+		body: JSON.stringify(data),
+	})
+		.then((response) => response.json())
+		.then((responseData) => {
+			localStorage.setItem('token', responseData.result);
+			localStorage.setItem('username', responseData.user.name);
+		})
+		.catch((error) => {
+			console.log(error);
```

Comment:
When you catch error it is better to use console.error to show error
```suggestion
			console.error(error);
```

Reply:
but there are no logs here in pr 
![image](https://github.com/YauheniAntsipenka/react_course/assets/77329785/2cbd0520-63c9-4451-b320-3deaebb6866f)